.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,13 @@
-Disregard the recommendations below if you use **Edit on Github** button to improve the docs in place.
+Disregard the recommendations below if you use **Edit on GitHub** button to
```

Comment:
don't make these templates according to regular rules. Github does not render them properly. It's better to leave to GH to decide.

Reply:
Oh I didn't mean to do that. Automatic pre-hook behavior. I just changed `h` for `H` in "GitHub". Reverted wrapping in 53f840c6 now.

Reply:
p.s. an example of why we shouldn't introduce automatic `git add` with `lint-staged` instead of `pretty-quick --bail` BTW. (Totally unrelated.)